smpe/bld/SMPMCS.txt

```diff
@@ -50,7 +50,9 @@
 ++SAMP(ZWEIKRT1)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWEIKRT2)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
 ++SAMP(ZWEIKRT3)     SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
-++SAMP(ZWEIMVS)      SYSLIB(SZWESAMP) DISTLIB(AZWESAMP) RELFILE(2) .
```

Comment:
IIRC we need to keep the old members around, or the SMP/e upgrades will fail. We can remove these with a clean fmid

Reply:
OK, I will keep the original names and create only one new.